bin/s2nc.c

```diff
@@ -411,6 +423,20 @@ int main(int argc, char *const *argv)
             early_data = load_file_to_cstring(optarg);
             GUARD_EXIT_NULL(early_data);
             break;
+        case OPT_PERF_SEND_AMOUNT:
```

Comment:
Why are we switching from character flags to number flags?

Reply:
this is so we don't have to specify a short flag. i didn't have time to go through and look for an unused character that matched.